core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/WebserviceSecurity.json

```diff
@@ -0,0 +1,139 @@
+
+            },
+            "wlst_attributes_path": "WP001",
+            "wlst_paths": {
+              "WP001": "/WebserviceSecurit${y:ies}/%WEBSERVICE%/WebserviceTokenHandler${:s}/%WSTOKENHANDLER%/ConfigurationPropert${y:ies}/%PROPERTY%"
+            }
+          }
+        },
+        "attributes": {
+          "ClassName":                           [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "ClassName",                          "wlst_path": "WP001", "value": {"default": "None"   }, "wlst_type": "string"            } ],
+          "HandlingOrder":                       [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "HandlingOrder",                      "wlst_path": "WP001", "value": {"default": 0        }, "wlst_type": "integer"           } ],
+          "Notes":                               [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "Notes",                              "wlst_path": "WP001", "value": {"default": "None"   }, "wlst_type": "string"            } ],
+          "TokenType":                           [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "TokenType",                          "wlst_path": "WP001", "value": {"default": "None"   }, "wlst_type": "string "           } ]
+        },
+        "wlst_attributes_path": "WP001",
+        "wlst_paths": {
+          "WP001": "/WebserviceSecurit${y:ies}/%WEBSERVICE%/WebserviceTokenHandler${:s}/%WSTOKENHANDLER%"
+        }
+      }
+    },
+    "attributes": {
+                "CompatibilityOrderingPreference":     [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "CompatibilityOrderingPreference",    "wlst_path": "WP001", "value": {"default": "None"   }, "wlst_type": "string"            } ],
+                "CompatibilityPreference":             [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "CompatibilityPreference",            "wlst_path": "WP001", "value": {"default": "None"   }, "wlst_type": "string"            } ],
+                "DefaultCredentialProviderSTSURI":     [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "DefaultCredentialProviderSTSURI",    "wlst_path": "WP001", "value": {"default": "None"   }, "wlst_type": "string"            } ],
+                "Notes":                               [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "Notes",                              "wlst_path": "WP001", "value": {"default": "None"   }, "wlst_type": "string"            } ],
+                "PolicySelectionPreference":           [ {"version": "[10,)",       "wlst_mode": "both",    "wlst_name": "PolicySelectionPreference",          "wlst_path": "WP001", "value": {"default": "NONE"   }, "wlst_type": "string"            } ]
+    },
+    "wlst_attributes_path": "WP001",
+    "wlst_paths": {
+        "WP001": "/WebserviceSecurit${y:ies}/%WEBSERVICE%"
+    }
+
+}
```

Comment:
Attributes need to use "default_value" and null value since aliases changes have been merged to main.
Pull the latest main and compare with other alias files.